### PR TITLE
fix(examples): Improve examples

### PR DESCRIPTION
**Pull request by zayKenyon**
**Please describe the changes this PR makes and why it should be merged:**
This PR aims to address the concerns brought up in #1147.

### CODE REVIEW

**Comment by isaacKenyon** [comment]
Should the spelling within the changes become Americanised?

**Comment by megatank58** [comment]
A command which asks questions doesn't seems like a good use case for something like autocomplete- the point of which is to provide dynamic options

**Review by almostSouji — CHANGES_REQUESTED** [review]
address lint warnings

File: guide/interactions/autocomplete.md [hunk 1]
```diff
@@ -66,16 +66,16 @@ To distinguish between multiple options, you can pass `true` into <DocsLink path
...
 
-	if (interaction.commandName === 'tag') {
+	if (interaction.commandName === 'guide') {
 		const focusedOption = interaction.options.getFocused(true);
 		let choices;
 
-		if (focusedOption.name === 'name') {
-			choices = ['faq', 'install', 'collection', 'promise', 'debug'];
+		if (focusedOption.name === 'query') {
+			const choices = ['Popular Topics: Threads', 'Sharding: Getting started', 'Library: Voice Connections', 'Interactions: Replying to slash commands', 'Popular Topics: Embed preview'];
```
Comment (almostSouji) [inline guide/interactions/autocomplete.md:74]: that const seems very wrong here

**Review by almostSouji — CHANGES_REQUESTED** [review]

**Review by almostSouji — APPROVED** [review]

**Pull request merged**